### PR TITLE
Add some rewrite rules

### LINKED ISSUE #24: Missing redirects
While doing a larger OpenSSL usability study, we found out that many online man pages that are linked from relevant Stack Overflow threads are inaccessible, since they miss a redirect (which is a pity!).

Specifically, many pages link to commands in this way:
* https://www.openssl.org/docs/apps/rsa.html

which has a redirect for 
* https://www.openssl.org/docs/manmaster/apps/rsa.html

However, in the `manmaster` section, the subsections are no longer `apps`/`ssl`/`crypto` but `man1`/`man3`/`man5`/`man7`.

Sure, it may not seem that important, but _half_ of the man page links from various internet tutorials our participants (~70 people) wanted to use got them a 404 on OpenSSL pages. Fixing would only require a handful of redirects. 

These rewrites would be needed (order matters, since the structure changed):

| From | To |
| ---- | ---- |
| /docs/manmaster/apps/config.html | /docs/manmaster/man5/config.html|
| /docs/manmaster/apps/x509v3_config.html   | /docs/manmaster/man5/x509v3_config.html|
| /docs/manmaster/crypto/bio.html | /docs/manmaster/man7/bio.html|
| /docs/manmaster/crypto/crypto.html | /docs/manmaster/man7/crypto.html|
| /docs/manmaster/crypto/ct.html | /docs/manmaster/man7/ct.html|
| /docs/manmaster/crypto/des_modes.html | /docs/manmaster/man7/des_modes.html|
| /docs/manmaster/crypto/evp.html | /docs/manmaster/man7/evp.html|
| /docs/manmaster/ssl/ssl.html | /docs/manmaster/man7/ssl.html|
| /docs/manmaster/crypto/x509.html | /docs/manmaster/man7/x509.html|
| /docs/manmaster/apps/* | /docs/manmaster/man1/*|
| /docs/manmaster/ssl/* | /docs/manmaster/man3/*|
| /docs/manmaster/crypto/* | /docs/manmaster/man3/*|

Of the overview manuals (`man7`), I couldn't find the counterparts of `Ed25519.html`, `RSA-PSS.html`, `X25519.html` and `ossl_store.html`, but I suspect these were newly written when moving to the `manX` system.

Another possibility is to redirect things like `/docs/apps/rsa.html` to a particular version of OpenSSL (e.g. `/docs/man1.1.0/apps/rsa.html`). This would now require the `apps`/`ssl`/`crypto` to `manX` translation, but I think it would be unwise to bind on a particular API version.

### PR DESCRIPTION
**Pull request by richsalz**
Recommended by a usability study conducted by Martin Ukrop at CRoCS, FI MU
Fixes #24

### CODE REVIEW

File: docs/.htaccess [hunk 1]
```diff
@@ -3,11 +3,20 @@ RewriteRule ^fips/fipsnotes.html /docs/fipsnotes.html [L,R=302,NC]
...
 RewriteRule ^misc/OCB-patent-grant-OpenSSL.pdf /source/OCB-patent-grant-OpenSSL.pdf [L,R=302,NC]
+RewriteRule ^manmaster/apps/config.html        /docs/manmaster/man5/config.html [L,R=302,NC]
+RewriteRule ^manmaster/apps/x509v3_config.html /docs/manmaster/man5/x509v3_config.html [L,R=302,NC]
+RewriteRule ^manmaster/crypto/bio.html /docs/manmaster/man7/bio.html [L,R=302,NC]
+RewriteRule ^manmaster/crypto/crypto.html      /docs/manmaster/man7/crypto.html [L,R=302,NC]
+RewriteRule ^manmaster/crypto/ct.html  /docs/manmaster/man7/ct.html [L,R=302,NC]
+RewriteRule ^manmaster/crypto/des_modes.html   /docs/manmaster/man7/des_modes.html [L,R=302,NC]
+RewriteRule ^manmaster/crypto/evp.html /docs/manmaster/man7/evp.html [L,R=302,NC]
+RewriteRule ^manmaster/ssl/ssl.html    /docs/manmaster/man7/ssl.html [L,R=302,NC]
+/docs/manmaster/crypto/x509.html        /docs/manmaster/man7/x509.html
```
Comment (levitte) [inline docs/.htaccess]: This line doesn't make any sense

**Review by levitte — CHANGES_REQUESTED** [review]

**Comment by richsalz** [comment]
new commit pushed to fix that editing error.

**Review by levitte — APPROVED** [review]
I'm not sure why master manpages would be accessed using the old structure, but I don't see he harm with this.

**Comment by richsalz** [comment]
Apparently some search engines have already crawled and cached the master docs with the old names.  :)  Squashed and merged, thanks.

**Pull request closed without merging**